Packs/MongoDB/Integrations/MongoDB/MongoDB.py

```diff
@@ -469,6 +534,31 @@ def pipeline_query_command(client: Client, collection: str, pipeline: str, limit
         return 'MongoDB: No results found', {}, raw_response
 
 
+def bulk_update_command(
+        client: Client,
+        collection: str,
+        filter: str,
+        update: str,
+        update_one=False,
```

Comment:
Since the `defaultValue` for this command in the yml it 'true', the update_one=False part is not really necessary. I'll change it to update_one=True for the code readability.